### PR TITLE
GEN-1192: house related moving flow

### PR DESCRIPTION
**Pull request by sladan-hedvig**
- House related moving flow
- Added apartment isStudent

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/MoveFlow/Sources/Model/MovingFlowModel.swift [hunk 1]
```diff
@@ -30,6 +66,15 @@ public struct MovingFlowModel: Codable, Equatable, Hashable {
         return quotes.first?.startDate ?? ""
     }
 
+    func maxNumberOfConsuredFor(_ type: HousingType) -> Int {
```
Comment (julia-andersson) [inline Projects/MoveFlow/Sources/Model/MovingFlowModel.swift:69]: typo, missing an "I". maxNumberOfCoInsured

File: Projects/MoveFlow/Sources/View/MovingFlowAddExtraBuildingView.swift [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+            value: vm.buildingType?.translatedValue ?? "",
+            placeholder: L10n.changeAddressExtraBuildingContainerTitle,
+            error: $vm.buildingTypeError
+        ) {
+            vm.showTypeOfBuilding()
+        }
+    }
+
+    private var livingArea: some View {
+        hFloatingTextField(
```
Comment (julia-andersson) [inline Projects/MoveFlow/Sources/View/MovingFlowAddExtraBuildingView.swift:58]: Don't we want m2 placeholder for this one as well?

**Review by julia-andersson — CHANGES_REQUESTED** [review]
https://github.com/HedvigInsurance/ugglan/assets/123949181/9ea29212-c70b-4931-b570-e9dad25de276

Not sure if you already are aware of this but the animations look off when switching between the screens

File: Projects/MoveFlow/Sources/View/MovingFlowAddExtraBuildingView.swift (hunk 2 above)
Comment (sladan-hedvig) [reply Projects/MoveFlow/Sources/View/MovingFlowAddExtraBuildingView.swift:58]: William was all for removing this m2

**Comment by sladan-hedvig** [comment]
> RPReplay_Final1696927204.MP4 
> Not sure if you already are aware of this but the animations look off when switching between the screens

Should be fine now :)

**Pull request merged**